doc/api/zlib.md

```diff
@@ -234,12 +285,15 @@ HTTP response to the client:
 ```js
 const zlib = require('zlib');
 const http = require('http');
+const { pipeline } = require('stream');
 
 http.createServer((request, response) => {
   // For the sake of simplicity, the Accept-Encoding checks are omitted.
   response.writeHead(200, { 'content-encoding': 'gzip' });
   const output = zlib.createGzip();
-  output.pipe(response);
+  pipeline(output, response, (err) => {
+    // Handle error.
```

Comment:
We can prefill this code as well.

Reply:
See previous comment. What do you recommend here given that the `writeHead()` has already been called and some amount of data may have already been written to the client